scripts/create_burndown.py

```diff
@@ -17,211 +17,402 @@
-  for i in range(len(new_tasks)):
-    if new_tasks[i] != 0:
-      effective_new_tasks_days.append(i - 0.25 + 1)
-      effective_new_tasks.append(new_tasks[i])
-  if len(effective_new_tasks) > 0:
-    plt.bar(effective_new_tasks_days, effective_new_tasks, .2, color='green')
-
-# Calculation of new story points
-if len(total_story_points) > 1:
-  new_story_points = [0]
-  for i in range(1, len(total_story_points)):
-    new_story_points.append(total_story_points[i] - total_story_points[i - 1])
-  effective_new_story_points_days = []
-  effective_new_story_points = []
-  for i in range(len(new_story_points)):
-    if new_story_points[i] != 0:
-      effective_new_story_points_days.append(i + 0.05 + 1)
-      effective_new_story_points.append(new_story_points[i])
-  if len(effective_new_story_points) > 0:
-    plt.bar(effective_new_story_points_days, effective_new_story_points, .2, color='black')
-
-# Calculation of new fast lane cards
-if len(total_fast_lane) > 1 and args.with_fast_lane:
-  new_fast_lane_cards = [0]
-  for i in range(1, len(total_fast_lane)):
-    new_fast_lane_cards.append(total_fast_lane[i] - total_fast_lane[i - 1])
-  effective_new_fast_lane_days = []
-  effective_new_fast_lane_cards = []
-  for i in range(len(new_fast_lane_cards)):
-    if new_fast_lane_cards[i] > 0:
-      effective_new_fast_lane_days.append(i + 0.1 + 1)
-      effective_new_fast_lane_cards.append(new_fast_lane_cards[i])
-  if len(effective_new_fast_lane_cards) > 0:
-    plt.bar(effective_new_fast_lane_days, effective_new_fast_lane_cards, .2, color='red')
-
-# Draw arrow showing already done tasks at begin of sprint
-tasks_done = burndown["days"][0]["tasks"]["total"] - burndown["days"][0]["tasks"]["open"]
-
-if tasks_done > 0 and not args.no_tasks:
-  plt.annotate("",
-	       xy=(x_days[0], scalefactor * max_story_points - 0.5 ), xycoords='data',
-	       xytext=(x_days[0], y_open_tasks[0] + 0.5), textcoords='data',
-	       arrowprops=dict(arrowstyle="<|-|>", connectionstyle="arc3", color='green')
-	      )
-
-  y_text = scalefactor * max_story_points
-  plt.text(0.7, y_text, str(int(tasks_done)) + " tasks done",
-	   rotation='vertical', verticalalignment='top', color='green'
-	  )
-
-if burndown["days"][0].has_key("tasks_extra") and not args.no_tasks:
-  tasks_done_extra = burndown["days"][0]["tasks_extra"]["done"]
-
-  plt.annotate("",
-	       xy=(x_days[0], scalefactor * y_story_points_done_extra[0] - 0.5 ), xycoords='data',
-	       xytext=(x_days[0], 0.5 - tasks_done_extra), textcoords='data',
-	       arrowprops=dict(arrowstyle="<|-|>", connectionstyle="arc3", color='green')
-	      )
-
-  y_text = - tasks_done_extra / 2
-  plt.text(0.4, y_text, str(int(tasks_done_extra)) + " extra",
-	   rotation='vertical', verticalalignment='center', color='green'
-	  )
-  plt.text(0.7, y_text, "tasks done",
-	   rotation='vertical', verticalalignment='center', color='green'
-	  )
 
+# Parse arguments and/or print help
+def argParser():
```

Comment:
I would name the function `parseCommandLine()`, this would make it clearer what happens.

The comment should not repeat what already is in the function name, but add information which is not explicitly there. In this case this would be the API documentation, i.e. how to use the function, when to call it, what it returns.